private_gpt/ui/ui.py

```diff
@@ -98,6 +98,8 @@ def __init__(
 
         self._selected_filename = None
 
+        self._response_style = True
+
```

Comment:
this variable name is very confusing, either make it explicit `response_streaming = True` or `response_style = "streaming"`

Reply:
Good point/suggestion. Changed checkbox to dropdown menu.